### PR TITLE
If there is only 1 module, automatically assign it to a cohort upon creation

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#1733** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1733?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/kotlin/com/terraformation/backend/accelerator/db/CohortStore.kt [hunk 1]
```diff
@@ -57,6 +64,8 @@ class CohortStore(
 
     cohortsDao.insert(row)
 
+    assignCohortModules(row.toModel())
+
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/accelerator/db/CohortStore.kt]: Could save `row.toModel()` to a variable and use it both here and in the `return` statement.

File: src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt [hunk 2]
```diff
@@ -360,6 +361,7 @@ abstract class DatabaseTest {
   protected val batchSubLocationsDao: BatchSubLocationsDao by lazyDao()
   protected val batchWithdrawalsDao: BatchWithdrawalsDao by lazyDao()
   protected val cohortsDao: CohortsDao by lazyDao()
+  protected val cohortModulesDao: CohortModulesDao by lazyDao()
```
Comment (sgrimm) [inline src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt:363]: Could move this up one line (M vs. s in alphabetical order).

**Review by sgrimm — APPROVED** [review]
Ship it! Had a couple comments you can pay attention to or not.

**Pull request merged**